yjit/src/codegen.rs

```diff
@@ -4982,7 +5002,10 @@ fn gen_struct_aref(
     cme: *const rb_callable_method_entry_t,
     comptime_recv: VALUE,
     _comptime_recv_klass: VALUE,
+    flags: u32,
```

Comment:
I had to pass these everywhere. Almost feels like we should have some struct for context? I don't know.